.husky/pre-commit

```diff
@@ -2,4 +2,3 @@
 . "$(dirname "$0")/_/husky.sh"
 
 yarn format
-git add .
```

Comment:
Just curiously asking is there any particular reason for removing this?

Reply:
Overall changes LGTM, though would love to hear from @Dhaiwat10.

Reply:
@Nazeeh21 This was adding every file change made to the commit, even when I was selecting just some to be staged.